### PR TITLE
1348 falsepositives xoslegacy

### PR DESCRIPTION
**Pull request by sq9mev**
## Pre-Request Checklist

- [ ] Passes rubocop code analysis (try `rubocop --auto-correct`)
- [ ] Tests added or adapted (try `rake test`)
- [x] Changes are reflected in the documentation
- [x] User-visible changes appended to [CHANGELOG.md](/CHANGELOG.md)

## Description

Add support for legacy Extreme devices, eg. Summit400-48t which marks their config as changed when `disable clipaging` is issued.

Closses issue #1348

### CODE REVIEW

**Comment by wk** [comment]
This looks reasonable. Is `ExtremeWare` separate from `XOS` in Extreme Networks nomenclature to a point that it makes sense including both in the supported OS list? Can we get some testing for this on both old and new models?

@sq9mev How many such devices have you run this against, and what level of variability in XOS versions do they have? Should additional testing be done before this is pulled?

**Comment by sq9mev** [comment]
> Is ExtremeWare separate from XOS in Extreme Networks nomenclature to a point that it makes sense including both in the supported OS list?
@wk I was wondering myself, see https://en.wikipedia.org/wiki/ExtremeXOS. Basing on this link, and discussion with @lubiewiatr i'm inclinded id deserves.

> Can we get some testing for this on both old and new models?
> How many such devices have you run this against, and what level of variability in XOS versions do they have? Should additional testing be done before this is pulled?

We have tens of Extremeware devices, most of them running Extremeware 7.8e.4, and few newer devices running xos 12.x - 15.x.

@lubiewiatr, could you provide us some feedback about how oxidized behaves with these devices?

**Comment by lubiewiatr** [comment]
@sq9mev I've updated our router.rb with about 70 extreme devices and so far everything looks correctly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Master
 
+* BUGFIX: xos model deos not modify config on legacy Extreme Networks devices (sq9mev)
```
Comment (wk) [inline CHANGELOG.md]: This looks like a typo (does?), also the sentence may be clearer if worded as 'no longer unintentionally modifies' or similar.

**Pull request merged**